web/src/user_group_edit.ts

```diff
@@ -1576,6 +1658,7 @@ export function setup_page(callback: () => void): void {
         );
         $groups_overlay_container.html(groups_overlay_html);
 
+        update_displayed_groups(selected_dropdown_filters);
```

Comment:
I think the current code, does not handle cases when group opened in right panel is not displayed on the left panel due to filters. Like mentioned in https://github.com/zulip/zulip/pull/32475/commits/637796aab99d015f1264f388c3c091b685a40074#r1948691380

Reply:
There is discussion for the expected behavior in https://chat.zulip.org/#narrow/channel/101-design/topic/display.20archived.20channels.20in.20the.20UI.

Reply:
Yes. Would be good to have a commit or maybe a separate PR which makes the behavior same for toggling between "Your groups" and "All groups" tabs.

Reply:
Also, does the current implementation follow this behavior -
> If you open the settings by some click, than the initially selected filter should be set so that includes the channel/group you're opening. I see that works now for clicking on a channel from the left sidebar, for instance, where they settings get set to the "Subscribed" filter. Clicking on an unsubscribed channel in the message view takes you to the "All channels" tab. We should make that work consistently, if there are situations where it doesn't.